bindings/terra/legion_terra_partitions.h

```diff
@@ -24,12 +24,13 @@
 //
 // ******************** IMPORTANT **************************
 
+#define LEGION_ENABLE_C_BINDINGS
+#include "legion.h"
```

Comment:
Same. In fact, I even wrote a comment about this above.

I'm going to stop commenting on this here, but this issue seems to recur throughout.